starter-code/src/functions-and-arrays.js

```diff
@@ -47,7 +91,17 @@ const wordsUnique = [
   'simple',
   'bring'
 ];
-
+function averageWordLength(mots){
+  for(var i=0;i<mots.length;i++){
+    let result=0
```

Comment:
`let result = [];` pour initaliser un array vide (le résultat de la fonction sera un array. 
Par ailleurs, il est préférable de l'initialiser avant la boucle for.